contrib/build-flatpak.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export LC_ALL=C
+set -e
+SECONDS=0
+
+repo_root="$(realpath $(dirname "$0")/..)"
```

Comment:
```suggestion
repo_root="$(realpath "$(dirname "$0")/..")"
```
nit (shellcheck warn)